data/420/552/245/420552245.geojson

```diff
@@ -21,38 +21,12 @@
     "mz:is_landuse_aoi":0,
     "mz:max_zoom":18.0,
     "mz:min_zoom":15.0,
-    "name:ces_x_preferred":[
-        "Pre\u0161erenovo n\u00e1m\u011bst\u00ed"
-    ],
-    "name:deu_x_preferred":[
-        "Pre\u0161erenplatz"
-    ],
     "name:eng_x_preferred":[
-        "St Marys Square"
-    ],
-    "name:epo_x_preferred":[
-        "Pre\u0161eren-placo"
-    ],
-    "name:eus_x_preferred":[
-        "Pre\u0161eren enparantza"
-    ],
-    "name:fin_x_preferred":[
-        "Pre\u0161erenin aukio"
-    ],
-    "name:fra_x_preferred":[
-        "Place Pre\u0161eren"
-    ],
-    "name:heb_x_preferred":[
-        "\u05db\u05d9\u05db\u05e8 \u05e4\u05e8\u05e9\u05e8\u05df"
-    ],
-    "name:jpn_x_preferred":[
-        "\u30d7\u30ec\u30b7\u30a7\u30fc\u30ec\u30f3\u5e83\u5834"
-    ],
-    "name:nld_x_preferred":[
-        "Pre\u0161erenplein"
+        "St Marys Park"
```

Comment:
This should match the `wof:name`, which has a `.` after the `St`. So this value should be `St. Marys Park`... though perhaps the Marys should be possessive? Place names are weird, please research and confirm.

Reply:
I see a mix of both "St. Marys Park" and "St. Mary's Park":

https://www.trulia.com/n/ca/san-francisco/st-marys-park/155139/
https://www.niche.com/places-to-live/n/st-marys-park-san-francisco-ca/
https://www.zillow.com/st.-marys-park-san-francisco-ca/
https://www.nytimes.com/2011/12/11/us/st-marys-park-in-san-francisco.html
http://www.stmaryspark.org/

Though most non-real estate sources use the apostrophe, so I've updated to match.